Makefile

```diff
@@ -31,10 +31,10 @@ npm_install: package.json
 frontend_build:
 	npm run --silent frontend-build:${GULP_ENVIRONMENT}
 
-test: show_environment test_pep8 test_python test_javascript
+test: show_environment test_flake8 test_python test_javascript
 
-test_pep8: virtualenv
```

Comment:
We don't even want an option on doing PEP8?  I guess not if they potentially conflict.